pkg/webhooks/workload_webhook.go

```diff
@@ -247,56 +190,19 @@ func validateTolerations(tolerations []corev1.Toleration, fldPath *field.Path) f
 		if len(toleration.Key) > 0 {
 			allErrors = append(allErrors, metav1validation.ValidateLabelName(toleration.Key, idxPath.Child("key"))...)
```

Comment:
what about this one?

Reply:
The rule needed to replace it would be as follows `size(self) <= 253 &&
self.matches('^([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]/?[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$')`. 

However, this validation is considered too expensive since we can't apply it directly to the `key` field of the Toleration. Hence, we got an `exceed cost estimate` message when implementing this rule.